### ISSUE TITLE
Time zone handling inconsistency breaks unit tests on Linux

### ISSUE THREAD

**Issue by nathanchere** [HTBox/allReady#2113]
```
Failed   AllReady.UnitTest.Areas.Admin.Features.Notifications.InitialRequestConfirmationsSentHandlerShould.UpdateRequestStatusToPendingConfirmation
Error Message:
 System.TimeZoneNotFoundException : The time zone ID 'Eastern Standard Time' was not found on the local computer.
---- System.IO.FileNotFoundException : Could not find file '/usr/share/zoneinfo/Eastern Standard Time'.
```

There is a library [TimeZoneConverter](https://github.com/mj1856/TimeZoneConverter) which looks like it might be relevant, but need to look into it further.

**Comment by nathanchere** [comment]
Have created a [.NET Fiddle example demonstrating this](https://dotnetfiddle.net/N3gdUu):

```using System;
using TimeZoneConverter;
					
public class Program
{
	public static void Main()
	{
		var result = TZConvert.WindowsToIana("Eastern Standard Time");
		Console.WriteLine(result);
	}
}```

Result: `America/New_York`

**Comment by nathanchere** [comment]
Related issue from dotnet core:

https://github.com/dotnet/corefx/issues/11897

**Comment by nathanchere** [comment]
This basically boils down to converting time zone descriptions to a `TimeZoneInfo` and back again. This is easy with the TimeZoneConverter library, basically by replacing calls to `TimeZoneInfo.FindSystemTimeZoneById` with `TZConvert.GetTimeZoneInfo`. Going back the other way should 'just work' as-is so long as all the `string`->`TimeZoneInfo` conversions are covered, but it's sloppy and ideally should be explicitly conform to either the Windows or IANA standard regardless of platform it's running on.

**Comment by nathanchere** [comment]
Closed by PR #2118
